benchmark/models/wideresnet/wideresnet.py

```diff
@@ -0,0 +1,142 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
```

Comment:
Can you use this header `2024 Tenstorrent AI ULC` for the new files that we add / change in 2024?

```suggestion
# SPDX-FileCopyrightText: © 2024 Tenstorrent AI ULC
```